src/modclub_assets/src/components/app/poh/POHConfirmationModal.tsx

```diff
@@ -73,6 +76,14 @@ const ConfirmationModal = ({
         type === "approve" ? { approved: null } : { rejected: null },
         rules
       );
+
+      // GTM: determine the amount of voted “human verification” tasks;
+      GTMManager.trackEvent("humanVerification", {
+        uId: hideStringWithStars(appState.loginPrincipalId),
```

Comment:
if we hideStr everywhere, we could put this logic inside GTMManager ?

Reply:
good suggestion, I thought about it, but analytics should not know what comes in the object, that’s why I chose this path, otherwise I’ll break the single responsiveness pattern